gem/backends.py

```diff
@@ -73,19 +73,25 @@ def _update_user_from_claims(user, claims):
                 try:
                     wagtail_group = Group.objects.get(name=group_name)
                     user.groups.add(wagtail_group)
+                    user.is_staff = True
+                    user.save()
```

Comment:
I think this needs to be in an if block. Otherwise you're setting it even if they are already staff and that seems like unnecessary db writes